File: BaekJoon/Greedy/1541.py
import sys


line = sys.stdin.readline().rstrip()
line = list(line.split("-"))
line = [list(map(int, list(x.split("+")))) for x in line ]

result = sum(line[0])

for i in range(1,len(line)):
    result-= sum(line[i])

print(result)

File: BaekJoonSwift/BaekJoonSwift/Greedy/1541.swift
import Foundation

var str = readLine()!

var temp = str.split(separator: "-")
var temp2 = [Int]()
temp2 = temp.map {
    var t = $0.split(separator: "+").map { Int($0)! }
    var sum = 0
    for a in t {
        sum += a
    }
    return sum
}

var result = temp2[0]

for i in 1..<temp2.count {
    result -= temp2[i]
}
print(result)
